src/chat/ListConversationScreen.tsx

```diff
@@ -17,16 +17,17 @@ export interface IListConversationProps {
 }
 
 export const ListConversationScreen: React.FC<IListConversationProps> = ({
-  hasSearchBar,
+  // hasSearchBar,
   onPress,
   renderCustomItem,
 }) => {
-  const { chatState, chatDispatch } = useChatContext();
+  const { chatDispatch } = useChatContext();
```

Comment:
create a file `selectors.ts` under `reducer`
create a function `getListConversation` receive a `state` and return `listConversation`